lib/features/email/presentation/email_controller.dart

```diff
@@ -89,9 +90,12 @@ class EmailController extends BaseController with AppLoaderMixin {
   EmailId? _currentEmailId;
   Identity? _identitySelected;
   List<EmailContent>? initialEmailContents;
-
   late Worker emailWorker;
-
+  final Map<EmailId, EmailLoaded> presentationEmailsLoaded = {};
```

Comment:
is it ok? I think we should have a mechanism to keep `loaded`. Fx: keep 10 recent mails, ....